ckanext/datagovcatalog/templates/robots.txt

```diff
@@ -0,0 +1,13 @@
+{% ckan_extends %}
+
+{% block all_user_agents -%}
+{{ super() }}
+Disallow: /dataset?
+Disallow: /organization?
+Disallow: /harvest?
+Disallow: /group?
+Sitemap: https://filestore.data.gov/gsa/catalog-next/sitemap/sitemap.xml
```

Comment:
This should be configurable. Can you double check the production.ini config in datagov-deploy? There might even already be a config value for this.

Reply:
This URL is 

`{{ ckanext.geodatagov.s3sitemap.aws_s3_url }} {{ ckanext.geodatagov.s3sitemap.aws_storage_path }}sitemap.xml`

https://filestore.data.gov/ + gsa/catalog-next/sitemap/ + sitemap.xml

Reply:
@adborden now the sitemap URL [use these settings](https://github.com/GSA/ckanext-datagovcatalog/pull/15/commits/e1e3e378a2a4e9646ed98ada7a3deecd9cbad3b9) (and we include [a test for this](https://github.com/GSA/ckanext-datagovcatalog/pull/15/commits/e8d9634ae4f1697c183182ad70f7a0321573dc8d))